### PR TITLE
New package: YoYoGames.GameMakerStudio version 2.3.3.574

### LINKED ISSUE #26187: Request for GameMaker: Studio 2
# Package Requested

- [x] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [x] I have performed a search and couldn't find this package.
- [ ] I think there is a new version available and I have provided the URL.

---

## Please provide the following information if you can:

**Publisher**: YoYo Games Ltd.

**Publisher Url**: https://yoyogames.com

**Package Name**: GameMaker: Studio 2

**Package Url**: https://www.yoyogames.com/gamemaker

**Description**: GameMaker Studio 2 is a complete development environment for making games. It comes with a comprehensive set of tools and lets you create games for Windows, Mac, Linux, Android, iOS, HTML5, Xbox, PlayStation, and Nintendo Switch. GameMaker has helped developers create much-loved and multi-million seller games such as: Undertale, Forager, & Hyper Light Drifter.

**Package Version**: 2.3.3.574

**Installer URL**: https://www.yoyogames.com/download/studio/2

## For new versions:

**Current Package Identifier**: tbd

**Additional notes:** GameMaker: Studio 2 has been given a free tier recently which allows the user to use the IDE without any limitations other than export options. The installer for the IDE's most recent update can be found on the Installer URL.

### PR DESCRIPTION
**Pull request by quhxl**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Resolves #26187

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26205)

### CODE REVIEW

File: manifests/y/YoYoGames/GameMakerStudio/2.3.3.574/YoYoGames.GameMakerStudio.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: YoYoGames.GameMakerStudio
+PackageVersion: 2.3.3.574
+PackageLocale: en-US
+Publisher: YoYo Games Ltd.
+PublisherUrl: https://www.yoyogames.com/en
+PublisherSupportUrl: https://help.yoyogames.com/hc/en-us
```
Comment (ItzLevvie) [inline manifests/y/YoYoGames/GameMakerStudio/2.3.3.574/YoYoGames.GameMakerStudio.locale.en-US.yaml:9]:
```suggestion
#PublisherSupportUrl: https://help.yoyogames.com/hc/en-us
```
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by quhxl** [comment]
When testing in the sandbox I got the error code 0x80070422.

**Comment by ItzLevvie** [comment]
Looks like it needs .NET Framework 3.5 - so we're stuck on dependency support:
image

**Comment by jedieaston** [comment]
Blocked for dependency support?

**Pull request closed without merging**